x/staking/keeper/keeper.go

```diff
@@ -63,14 +63,12 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 }
 
 // Set the validator hooks
-func (k *Keeper) SetHooks(sh types.StakingHooks) *Keeper {
```

Comment:
This pattern makes little sense to me give the name `SetHooks`.

Reply:
I think we should add an API-breaking changelog entry, iirc Osmosis is using SetHooks pretty heavily